sources/packages/web/src/components/aest/students/assessment/Request.vue

```diff
@@ -10,59 +10,71 @@
         >
         </body-header>
         <content-group class="mt-4">
-          <DataTable
-            :value="requestedAssessment"
-            :paginator="true"
-            :rows="DEFAULT_PAGE_LIMIT"
-            :rowsPerPageOptions="PAGINATION_LIST"
-            :totalRecords="requestedAssessment.length"
-          >
-            <template #empty>
-              <p class="text-center font-weight-bold">No records found.</p>
-            </template>
-            <Column
-              field="submittedDate"
-              header="Submitted date"
-              sortable="true"
-              ><template #body="slotProps">{{
-                dateOnlyLongString(slotProps.data.submittedDate)
-              }}</template></Column
+          <toggle-content :toggled="!requestedAssessment.length">
+            <DataTable
+              :value="requestedAssessment"
+              :paginator="true"
```

Comment:
Is "true" a variable that is defined somewhere like the other ones (:rows="DEFAULT_PAGE_LIMIT", :rowsPerPageOptions="PAGINATION_LIST", etc) ?

Reply:
The change is not actually from this PR but what was the question here? Sorry, I am not following?

Reply:
Is it about using `:paginator="true"` vs `paginator="true"`?
If this is the case, the first one is a boolean value assigned to a boolean property, the second one is a string value assigned to a boolean property, that is my understanding.